examples/test_api.c

```diff
@@ -185,11 +190,34 @@ int test_abtoull(char* str, unsigned long long* val)
   return SCR_SUCCESS;
 }
 
-double getbw(char* name, char* buf, size_t size, int times)
+size_t get_my_file_offset()
+{
+  uint64_t file_offset = 0;
+
+  if (use_shared_file) {
+    uint64_t send_buf;
+
+    if (rank == 0) {
+      send_buf = 0;
+    }
+    else {
+      send_buf = my_filesize;
+    }
+
+    int count = 1;
+    MPI_Scan(&send_buf, &file_offset, count, MPI_UINT64_T, MPI_SUM, MPI_COMM_WORLD);
```

Comment:
I think this scan code might be off a bit.  I think you might want something like:
```suggestion
    uint64_t send_buf = my_filesize;
    MPI_Scan(&send_buf, &file_offset, 1, MPI_UINT64_T, MPI_SUM, MPI_COMM_WORLD);
    file_offset -= my_filesize;
```

Reply:
yeah.  good catch